tapir/core/tapir_email_base.py

```diff
@@ -106,6 +106,15 @@ def send_to_tapir_user(self, actor: User | None, recipient: TapirUser):
             tapir_user=recipient,
         )
 
+    def to_log(self, email, actor, tapir_user, share_owner):
+        EmailLogEntry().populate(
```

Comment:
We should prepare for other emails that should not have the body saved in the log. 
We could have an "abstract" method in `TapirEmailBase .does_include_email_body_in_log_entry()`, that all child classes implement and return just True or False. Then in `to_log`, we could call either `EmailLogEntry().populate` or a new `EmailLogEntry().populate_without_body` depending on `does_include_email_body_in_log_entry`